src/alchemlyb/postprocessors/units.py

```diff
@@ -0,0 +1,122 @@
+'''Unit conversion and constants.'''
+
+from scipy.constants import R, calorie
+
+kJ2kcal = 1 / calorie
+R_kJmol = R / 1000
```

Comment:
add doc as in
```suggestion
#: conversion factor from kJ to kcal, based on :data:`scipy.constants.calorie`
kJ2kcal = 1 / calorie

#: Gas constant :math:`R` in kJ/(mol K), based on :data:`scipy.constants.R`
R_kJmol = R / 1000
```

I like your approach here.
```

Reply:
Also add these constants to the documentation somewhere
```reST
.. autodata:: kJ2kcal
.. autodata:: R_kJmol
```